auto_round/autoround.py

```diff
@@ -179,6 +179,66 @@ def __new__(
 
 @deprecated("AutoRound")
 class AutoRoundLLM(AutoRound):
```

Comment:
I’m wondering why it inherits from AutoRound instead of the base compressor. Since AutoRound has now been changed to a router, it no longer fits as a base class.